src/components/TransactionTable.vue

```diff
@@ -464,6 +465,11 @@ function getValueDisplay(value: string) {
     }
 }
 
+// quasar override
+.q-table__middle{
+    @include scroll-bar;
+}
+
```

Comment:
Can you move this style to the end of the file src/css/app.scss ?
We already have some quasar overrides there. It would be nice to have them all there in the same place.